### DISCUSSION TITLE
No repairing necessary even with Z2M Edge?

### DISCUSSION THREAD

**Discussion by Stooovie** [Koenkk/zigbee2mqtt#8839] [General]
Hi, I'm using the Edge branch (for compatibility with a TS011F plug). I'd like to switch from CC2531 to CC2652 (Slaesh's). I'm seeing in the Z2M FAQ that beginning with 1.21.0, re-pairing is not necessary.

Does that apply to a Edge branch installed today (21.9.2021)? Thanks!

**Comment by Koenkk** [comment]
> Does that apply to a Edge branch installed today (21.9.2021)? Thanks!


yes
